jetty-slf4j-impl/src/test/java/org/eclipse/jetty/logging/JMXTest.java

```diff
@@ -35,18 +40,24 @@ public void testJMX() throws Exception
     {
         MBeanServer mbeanServer = ManagementFactory.getPlatformMBeanServer();
 
-        Properties props = new Properties();
-        JettyLoggerConfiguration config = new JettyLoggerConfiguration(props);
-        JettyLoggerFactory loggerFactory = new JettyLoggerFactory(config);
-
         ObjectName objectName = ObjectName.getInstance("org.eclipse.jetty.logging", "type", JettyLoggerFactory.class.getSimpleName().toLowerCase(Locale.ENGLISH));
-        mbeanServer.registerMBean(loggerFactory, objectName);
+        mbeanServer.registerMBean(LoggerFactory.getILoggerFactory(), objectName);
+
+        // Verify MBeanInfo
+        MBeanInfo beanInfo = mbeanServer.getMBeanInfo(objectName);
+
+        MBeanAttributeInfo[] attributeInfos = beanInfo.getAttributes();
+        assertThat("MBeanAttributeInfo count", attributeInfos.length, is(2));
+
+        MBeanAttributeInfo attr = Stream.of(attributeInfos).filter((a) -> a.getName().equals("LoggerNames")).findFirst().orElseThrow();
+        assertThat("attr", attr.getDescription(), is("List of Registered Loggers by Name."));
 
         JettyLoggerFactoryMBean mbean = JMX.newMBeanProxy(mbeanServer, objectName, JettyLoggerFactoryMBean.class);
 
         // Only the root logger.
         assertEquals(1, mbean.getLoggerCount());
```

Comment:
This is testing the attribute via the JMX proxy. We need to test the operation (a call to `setLoggerLevel()`) too.

Reply:
Fixed.